programs/solana-nft-anchor/src/lib.rs

```diff
@@ -8,8 +8,8 @@ use anchor_spl::{
     token::{mint_to, Mint, MintTo, Token, TokenAccount},
 };
 use mpl_token_metadata::{
-    pda::{find_master_edition_account, find_metadata_account},
-    state::DataV2,
+    accounts::{MasterEdition, Metadata as MetadataAccount},
+    types::DataV2,
```

Comment:
```suggestion
```

Reply:
@rosarp, for the removed `types::DataV2,`, could you reimport it from the `anchor_spl` module 

i.e 

```rs
use anchor_spl::{
    metadata::{
        mpl_token_metadata::types::DataV2, // import it here instead
    },
};
```